### DISCUSSION TITLE
New tabs

### DISCUSSION THREAD

**Discussion by devnyxie** [MrGreensWorkshop/MrGreen-JekyllTheme#102] [Feature requests]
### Is your feature request related to a problem?

I think that many people would need Gallery, Skills or other tabs.. They don't have to be enabled by default, just to "exist" in _layouts 🤔

### Describe the solution you'd like

Personally I don't know sh bout ruby, but if you don't mind I'll help as I can and learn in the process:3

**Comment by devnyxie** [comment]
fa-address-book-o -> biography
fa-camera-retro -> gallery
      fa-folder  -> folders in gallery(?)
fa-terminal/fa-book/fa-coffee/fa-code  -> skills (commented multiple choice I guess, not all users are programmers)

**Comment by MrGreensWorkshop** [comment]
Hi, if you are referring this code, it's liquid.
```liquid
{%- include util/auto-content-generator.liquid -%} {{ website_info_text_first }}
```
You don't need any ruby knowledge for this project. 
Only one file related with ruby is [Gemfile](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/4cfccef082c3223245597afe59593c2cec7610b2/Gemfile).

Check this for [JekyII / Liquid](https://jekyll-theme-mrgreen-demo.mrgreensworkshop.com/tabs/links.html) links

Check https://jekyllrb.com/docs/ for _layouts and etc.

Currently I don't have any plan to add those layout pages. 

for adding biography check for privacy-policy.md and its layout /_layouts/privacy-policy.md 
for adding gallery I think some CSS will be required.

If you want to contribute, as always PR's are welcome.
If you check other tab pages I think you can figure it out. It's not that hard. 

for further question please use [discussions](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/discussions/)

I think that's all I can say.
